src/Common/src/Interop/Windows/sspicli/SSPIWrapper.cs

```diff
@@ -217,100 +155,54 @@ public static SafeFreeCredentials AcquireCredentialsHandle(SSPIInterface secModu
-            }
-#endif
+            NetEventSource.Exit(null, outCredential);
             return outCredential;
         }
 
         internal static int InitializeSecurityContext(SSPIInterface secModule, ref SafeFreeCredentials credential, ref SafeDeleteContext context, string targetName, Interop.SspiCli.ContextFlags inFlags, Interop.SspiCli.Endianness datarep, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, ref Interop.SspiCli.ContextFlags outFlags)
         {
-            if (SecurityEventSource.Log.IsEnabled())
-            {
-                SecurityEventSource.Log.InitializeSecurityContext(credential.ToString(),
-                    LoggingHash.ObjectToString(context),
-                    targetName,
-                    inFlags);
-            }
+            NetEventSource.Log.InitializeSecurityContext(credential, context, targetName, inFlags);
 
             int errorCode = secModule.InitializeSecurityContext(ref credential, ref context, targetName, inFlags, datarep, inputBuffer, outputBuffer, ref outFlags);
 
-            if (SecurityEventSource.Log.IsEnabled())
-            {
-                SecurityEventSource.Log.SecurityContextInputBuffer(nameof(InitializeSecurityContext), (inputBuffer == null ? 0 : inputBuffer.size), outputBuffer.size, (Interop.SECURITY_STATUS)errorCode);
-            }
-
+            NetEventSource.Log.SecurityContextInputBuffer(nameof(InitializeSecurityContext), inputBuffer?.size ?? 0, outputBuffer.size, (Interop.SECURITY_STATUS)errorCode);
+            
             return errorCode;
         }
 
         internal static int InitializeSecurityContext(SSPIInterface secModule, SafeFreeCredentials credential, ref SafeDeleteContext context, string targetName, Interop.SspiCli.ContextFlags inFlags, Interop.SspiCli.Endianness datarep, SecurityBuffer[] inputBuffers, SecurityBuffer outputBuffer, ref Interop.SspiCli.ContextFlags outFlags)
         {
-            if (SecurityEventSource.Log.IsEnabled())
-            {
-                SecurityEventSource.Log.InitializeSecurityContext(credential.ToString(),
-                    LoggingHash.ObjectToString(context),
-                    targetName,
-                    inFlags);
-            }
+            NetEventSource.Log.InitializeSecurityContext(credential, context, targetName, inFlags);
 
             int errorCode = secModule.InitializeSecurityContext(credential, ref context, targetName, inFlags, datarep, inputBuffers, outputBuffer, ref outFlags);
 
-            if (SecurityEventSource.Log.IsEnabled())
-            {
-                SecurityEventSource.Log.SecurityContextInputBuffers(nameof(InitializeSecurityContext), (inputBuffers == null ? 0 : inputBuffers.Length), outputBuffer.size, (Interop.SECURITY_STATUS)errorCode);
-            }
+            NetEventSource.Log.SecurityContextInputBuffers(nameof(InitializeSecurityContext), inputBuffers?.Length ?? 0, outputBuffer.size, (Interop.SECURITY_STATUS)errorCode);
```

Comment:
There's very little work happening in the call site; didn't seem necessary to put it behind a guard.  But I can if you'd like.